sdk/src/functions/connectFactory.ts

```diff
@@ -11,13 +12,24 @@ export interface ConnectParams {
 
 type ConnectedAccount = Account & { isReady: () => Promise<Account> };
 
-export const connect = (): Promise<ConnectedAccount> =>
-  connectFactory({ embedManager: EmbedManager.getInstance() })();
+export const connect = (
+  networkId: string,
+  chainId: ChainId,
+): Promise<ConnectedAccount> =>
+  connectFactory({ embedManager: EmbedManager.getInstance() })(
+    networkId,
+    chainId,
+  );
 
 export const connectFactory =
   ({ embedManager, timeout = 5 * 60 * 1000 }: ConnectParams) =>
-  async (): Promise<ConnectedAccount> => {
+  async (networkId: string, chainId: ChainId): Promise<ConnectedAccount> => {
+    const connectParams = new URLSearchParams({
+      networkId,
+      chainId,
+    });
     embedManager.openSidebar();
+    embedManager.setSidebarPath(`/embedded/sidebar#${connectParams.toString()}`);
```

Comment:
We might want to do this the other way around, set the path first (so it can load 5ms earlier 🙃 ), THEN show the sidebar